### ISSUE TITLE
Side bar not interact-able with dialog

### ISSUE THREAD

**Issue by Kitenite** [GoogleChromeLabs/ProjectVisBug#622]
Thank you for the awesome work on the popover, this is a huge unlock!

I notice from testing that the side bar is still not interact-able when a dialog is open. Seems like Chrome is also capturing the click on the dialog even if it's not the top item in the top-layer. What do you think could be going on? 

https://github.com/GoogleChromeLabs/ProjectVisBug/assets/31864905/afc9e47d-3bba-45e7-b837-0358e0d7ca29

**Comment by argyleink** [comment]
Looks to me like VisBug was opened before the Dialog, and so it's behind the dialog which ALSO has made the rest of the page `inert`. Curious, if you launch VisBug after the dialog is open, is the issue the same (probably still inert).

**Comment by Kitenite** [comment]
Yes, Visbug was opened after opening the dialog. Here's the top layer. 

![Screenshot 2024-05-02 at 5 09 49 PM](https://github.com/GoogleChromeLabs/ProjectVisBug/assets/31864905/33be40f5-6c3e-4cc1-92f8-8aa311b6e0d1)

**Comment by argyleink** [comment]
hmm, i just tested on 2 different `<dialog>` demo's I've made, and I can't recreate what you're experiencing?
![Screenshot 2024-05-02 at 2 15 55 PM](https://github.com/GoogleChromeLabs/ProjectVisBug/assets/1134620/6dd3e9ad-fc18-4571-a935-e3a33010e179)
![Screenshot 2024-05-02 at 2 18 32 PM](https://github.com/GoogleChromeLabs/ProjectVisBug/assets/1134620/ea9f576a-b24e-4883-805c-bba2e06e2557)

**Comment by Kitenite** [comment]
Maybe a daisyui thing but it's not working for their modal
https://daisyui.com/components/modal/

**Comment by argyleink** [comment]
oh i see! you can interact with the modal from visbug but not change tools. i can change the visbug tool via the hotkey, but can't click it. 

attempts to click focus the modal too, def making me think this is about `inert`

![Screenshot 2024-05-02 at 3 18 36 PM](https://github.com/GoogleChromeLabs/ProjectVisBug/assets/1134620/a064e0f2-9a2b-4657-9ce7-256dfb701cb0)

hmmm, i'll ponder some workarounds, see if there's a way to keep visbug interactive. thanks for loggin this!

**Comment by Kitenite** [comment]
Np, thanks for pushing the update! I didn't know about `inert`, will take a look too.
